.gitignore

```diff
@@ -66,4 +66,3 @@ lokinet-bootstrap.exe
 regdbhelper.dll
 # xcode
 xcuserdata/
-
```

Comment:
extraneous change

Reply:
yeah added stuff to gitignore then removed it later